tests/CocktailParty.tex

```diff
@@ -9,46 +9,45 @@ \subsection{Focus}
-	\item \textbf{Barman:} The Barman bay be standing either behind the bar or next to it, depending on the arena setup.
+	\item \textbf{Guests:} At least five people are distributed in a predefined \quotes{party room} either sitting or standing.
+                Three of the guests have drink orders.
+	\item \textbf{Bar:} The bar is any flat surface where objects can be placed, in a room other than the \quotes{party room}.
+                All available beverages are on top of the bar.
+	\item \textbf{Barman:} The Barman may be standing either behind the bar or next to it, depending on the arena setup.
 \end{itemize}
 
 \subsection{Task}
 
 \begin{enumerate}
 
-	\item \textbf{Entering:} The robot enters the arena and navigates to the party room and waits for being called.
-
+	\item \textbf{Entering:} The robot enters the arena and navigates to the party room.
 	\item \textbf{Getting called:} The guests call the robot simultaneously, either rising an arm, waving, or shouting. The robot has to approach one of them.
-	% Remark: Themself is the correct word used instead of himself/herself  when person gender is unknown.
-	The calling person introduces themself by name before giving the order of a drink. The robot leads the dialogue to learn the person and retrieve their drink order. \\
-
-	The robot can decide to skip the detection of the calling and ask one person to walk in front of it. In this case, the referees determine the person to approach the robot.
+        Optionally, the robot can skip the call detection and ask for a person to step in front of it (the referees determine who approaches to the robot).
 
-	\item \textbf{Taking the order:} After the robot has fetched the order of the first guest, it can either fetch more orders (i.e.~ find next calling guest or looking for the sitting one) or proceed to place the order. In the first case, the robot searches for the remaining calling people. During the search process, the robot is allowed to either ask people to call for it again, or to ask people to come to it and to give a new order. In both cases the robot may call into the room.
+	The calling person introduces themself by name before giving the order of a drink.
+	The robot leads the dialogue to learn the person and retrieve their drink order. \\
 
-	\item \textbf{Sitting person:} At least one person is sitting and minding their own business (i.e. not looking directly to the robot). The robot must locate that person and ask if the person wants something to drink. The robot must also ask for the person's name and memorize them (i.e. execute a learning procedure of the name and the person's features). \\
 
-	\textbf{Remark:} At least one sitting person has an order to place. No sitting person will attend to the robot's call
-	(i.e. robot has to approach to them).
+	\item \textbf{Taking the order:} After the robot has took the order of the first guest, it can either take more orders or proceed to place the order.
```

Comment:
\item \textbf{Taking the order:} After the robot has taken the order of the first guest, it can either take more orders or proceed to place the order.